react/src/components/ResourceAllocationFormItems.tsx

```diff
@@ -239,9 +243,9 @@ const ResourceAllocationFormItems: React.FC<
     ]);
     // If the current accelerator type is not available,
     // change accelerator type to the first supported accelerator
-    const nextAcceleratorType = acceleratorSlots?.[currentAcceleratorType]
+    const nextAcceleratorType = acceleratorSlotsInRG?.[currentAcceleratorType]
       ? currentAcceleratorType
-      : _.keys(acceleratorSlots)[0];
+      : _.keys(acceleratorSlotsInRG)[0];
```

Comment:
```suggestion
      : _.first(_.keys(acceleratorSlotsInRG));
```
If acceleratorSlotsInRG is undefined, it occurs error.